templates/layout/app.css

```
@@ -0,0 +1,565 @@
+.codeblock { position: relative; margin: 10px 0 0; }
+.codeblock pre {
+    margin: 0;
+    background: var(--bg-sunk);
+    border: 1px solid var(--rule);
+    border-radius: var(--radius-sm);
+    padding: 12px 46px 12px 14px;
+    overflow-x: auto;
+    font-size: 12.5px;
+    line-height: 1.6;
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)